embedding/csharp/Tizen.Flutter.Embedding/Tizen.Flutter.Embedding.csproj

```diff
@@ -28,7 +28,7 @@
     <PackageReference Include="Tizen.NET" Version="9.0.0.16760" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'tizen40'">
+  <ItemGroup Condition="'$(TargetFramework)' == 'tizen40' Or '$(TargetFramework)' == 'tizen80'">
```

Comment:
I have a question.
Why didn't you check 'tizen90'?
I asked because the TargetFrameworks above also included tizen90.

Reply:
As mentioned in [the comment above](https://github.com/flutter-tizen/flutter-tizen/pull/569#discussion_r1823768904), NUIFlutterView.cs is a feature for Tizen 6.5. Tizen 6.5 is using tizen90. https://github.com/Samsung/TizenFX

Reply:
I understood. Thanks for explaining.